### ISSUE TITLE
[Feature] Showing the presentation from Github directly

### ISSUE THREAD

**Issue by mantzas** [fsprojects/FsReveal#109]
Hi,

it would be nice if we could use a online service, point to a github repository and see the presentation rendered without having to generate or host it.

**Comment by isaacabraham** [comment]
A kind of FsReveal-as-a-service, if you will. Nice idea.

**Comment by mantzas** [comment]
exactly. This was a nice feature of the go package present where you give you can have a service the you can point in the Url to the Github repo you own and have the service show you the presentation.

example:

http://go-talks.appspot.com/github.com/mantzas/presentations/20161106_shaken_to_the_dotnet_core/shaken_to_the_dotnet_core.slide#1

where http://go-talks.appspot.com is the service and github.com/mantzas/presentations/20161106_shaken_to_the_dotnet_core/shaken_to_the_dotnet_core.slide#1 is the location of the presentation.

With this you can share a presentation online as a link!

**Comment by isaacabraham** [comment]
If someone wants to write a simple app that can do this - perhaps a Suave app that takes in the repo, creates a paket dependencies that contains the GH repo, does an paket install and kicks off the presentation within suave? I'd be happy to host such an app in Azure.

**Comment by pirrmann** [comment]
Actually it just needs to be built, doesn't it? It can then be pushed to github pages, you don't need to host the presentation. If there is no such fake target in FsReveal, adding one would be a good start in my opinion.

**Comment by mantzas** [comment]
@pirrmann yes, this could be another way to do it. But i have a repo that contains all my presentations. Not all are written in FsReveal, some are in go's present package. it would be a issue to have one ph pages for this repo with all presentations.
